### ISSUE TITLE
[Edition] Text Edition Bug. 

### ISSUE THREAD

**Issue by louiesun** [SVG-Edit/svgedit#958]
**Describe the bug**
Move the text to the left-bottom corner automaticly when editing some svg

**To Reproduce**
1. Open the given svg. 
2. Try to edit the text. 
3. See error. 
![out-0001](https://github.com/SVG-Edit/svgedit/assets/95512329/ecee8edf-5727-4eff-8892-ff02e03b418a)

**Expected behavior**
Normal edition

**SVG-Edit environment (IMPORTANT)**
The Latest PreView. And the same action. 
[Try SVGEdit V7 (master branch on github)](https://svgedit.netlify.app/editor/index.html)
[Try SVGEdit V7 (latest published version on npm)](https://unpkg.com/svgedit@latest/dist/editor/index.html)

**Desktop (please complete the following information):**
 - Browser Chromium
 - Version 109.0.5414.120 (Win.7)

**Screenshots**
[The latest npm version. ](https://unpkg.com/svgedit@7.3.3/dist/editor/index.html)
![ss](https://s3.bmp.ovh/imgs/2024/04/16/00b6ddecb2b39372.gif)

**Log**
If applicable, copy any error logs in your browser console. This can be very
helpful in identifying the problem.

**Additional context**
The svg is generated by MuPDF/PyMuPDF from a PDF file. 
Maybe the `<tspan>`？

**Comment by jfhenon** [comment]
This is indeed a bug that seems to be linked to the <tspan>

If you load this svg:
```<svg width="640" height="480" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
 <!-- Created with SVG-edit - https://github.com/SVG-Edit/svgedit-->
 <g class="layer">
  <title>Layer 1</title>
  <text id="svg_1" x="100" y="100">This is
   <tspan id="svg_2">a bug!</tspan></text>
 </g>
</svg>
```

You can move it using the arrow key or the mouse and it will create the bug

(jnterestingly, moving using the top bar x/y buttons seems to work normally)

A PR to fix this would be welcome.

**Comment by jfhenon** [comment]
fixed by #1006
